syft/formats/spdxtagvalue/test-fixtures/snapshot/TestSPDXJSONSPDXIDs.golden

```diff
@@ -8,10 +8,17 @@ Creator: Organization: Anchore, Inc
 Creator: Tool: syft-v0.42.0-bogus
 Created: redacted
 
+##### Package: foobar/baz
+
+PackageName: foobar/baz
+SPDXID: SPDXRef-DocumentRoot-Directory-foobar-baz
+PrimaryPackagePurpose: FILE
+FilesAnalyzed: false
+
 ##### Package: @at-sign
 
 PackageName: @at-sign
-SPDXID: SPDXRef-Package---at-sign-3732f7a5679bdec4
+SPDXID: SPDXRef-Package--at-sign-3732f7a5679bdec4
```

Comment:
I see that this got better from `---` to `--` but is why is there the extra `-` here? (this might be out of scope for this PR, so consider this non-blocking)

Reply:
It is because there is an `@` character, which is changed to `-` due to SPDX ID rules rather than just removing the character. This behavior has existed for a while, so I think it's out of scope.

Reply:
ah! thanks